src/components/ProjectStatusDek/index.jsx

```diff
@@ -17,49 +18,31 @@ import {
  * @returns {JSX.Element}
  */
 export default function ProjectStatusDek({ id }) {
-  const status = atoms.useRecoilValue(
-    atoms.status(id),
-  );
-
-  const lastUploaded = atoms.useRecoilValue(
-    atoms.projectLastUploaded(id),
-  );
+  const pubStatus = atoms.useRecoilValue(atoms.projectPublishedStatus(id));
+  const lastUpdated = atoms.useRecoilValue(atoms.projectLastUpdated(id));
 
   const dekClass = cls(styles.dek, type.textSm);
 
-  if (status === 'archive') {
-    return <div className={dekClass}>Download to start editing</div>;
-  }
-
-  const renderText = () => {
-    // TODO: Status rework
-    switch (status) {
-      case STATUS_PROJECT_OK:
-        return 'Last backed up: ';
-      case STATUS_PROJECT_VALID_UPLOAD:
-        return 'Not backed up: ';
+  const statusText = useMemo(() => {
+    switch (pubStatus) {
+      case STATUS_PROJECT_DRAFT:
+        return 'This project has not been published';
+      case STATUS_PROJECT_PUBLISHED:
+        return 'Last published: ';
+      case STATUS_PROJECT_CHANGES:
+        return 'Change since last publish: ';
```

Comment:
This line is too long when there's a relative timestamp.
```suggestion
        return 'Changes since publish: ';
```